backend/src/main/java/com/bakdata/conquery/models/error/ErrorMessages.java

```diff
@@ -0,0 +1,64 @@
+package com.bakdata.conquery.models.error;
+
+import java.sql.SQLException;
+
+import c10n.annotations.De;
+import c10n.annotations.En;
+import com.bakdata.conquery.models.forms.util.Alignment;
+import com.bakdata.conquery.models.forms.util.Resolution;
+import com.bakdata.conquery.models.query.entity.Entity;
+
+public interface ErrorMessages {
+
+	@En("An unknown error occurred")
+	@De("Ein unerwarteter Fehler ist aufgetreten.")
+	String unknownError();
+
+	@En("An unknown error occurred, while creating the query.")
+	@De("Ein unerwarteter Fehler ist aufgetreten, beim erzeugen der Anfrage.")
+	String executionCreationUnspecified();
+
+	@En("Could not find an ${1} element called `${0}`")
+	@De("Es konnte kein ${1} Objekt mit der Id `${0}` gefunden werden.")
+	String executionCreationResolve(String id, String clazz);
+
+	@En("There are ${0} columns in the format but ${1} in at least one row")
+	@De("Das Format gibt ${0} Spalten vor, eine Zeile enthält aber ${1} Spalten.")
```

Comment:
Do you need to keep in mind the singular case here? Could happen quite often when a user doesn't get column seperator right